comp/pits/helpdesk/expert-list.yaml

```diff
@@ -0,0 +1,24 @@
+summary: Create domain expert list
+
+priority: major
+
+component: Competition
+
+milestone: $SRYYYY Competition
+
+description: >-
+  If there is a problem at helpdesk, that those on helpdesk cannot solve, they need to know who to escalate to.
+
+  There will be different experts depending on the domain.
+
+  Example domains:
+
+  - Rules
```

Comment:
```suggestion
  - Rules (probably whoever the head judge is)
```

Reply:
It won't _just_ be the head judge, and there may be a number of people who know things about different parts of the rules (eg regulations vs game design). Leaving this unspecified forces the person to think about who may be best.